<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Occam2DMTModel</class>
 <widget class="QWidget" name="Occam2DMTModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Occam2DMT</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,10,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_newModel">
       <property name="text">
        <string>新建模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addrow">
       <property name="text">
        <string>增加行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delrow">
       <property name="text">
        <string>删除行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_addcol">
       <property name="text">
        <string>增加列</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delcol">
       <property name="text">
        <string>删除列</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_expand">
       <property name="text">
        <string>扩边</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_value">
       <property name="text">
        <string>赋值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_importViewPoints">
       <property name="text">
        <string>导入测点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_import">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_export">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_sure">
       <property name="text">
        <string>完成</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_paint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget_color">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_value">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_colorScale">
       <property name="text">
        <string>设置色标</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_RhoRange">
       <property name="text">
        <string>设置电阻率范围</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
